installer/pkg/config-generator/fixtures/kube-system.yaml

```diff
@@ -22,6 +22,20 @@ data:
       service_cidr: 10.3.0.0/16
     routingConfig:
       subdomain: test.cluster.com
+  mao-config: |
+    apiVersion: v1
+    aws:
+      availabilityZone: ""
+      clusterID: ""
+      clusterName: test
+      containerLinuxVersion: 1800.7.0
```

Comment:
We are dropping support for Container Linux very soon. Why is there a direct dependency on CL here?

Reply:
I'm actually not sure... @enxebre?